### ISSUE TITLE
Spongeforge with Evilcraft

### ISSUE THREAD

**Issue by hxpx** [rubensworks/EvilCraft#316]
It's dos'nt work crashing when trying to load evilcraft .There're crash log with spongeforge-1.8-1577-2.1-DEV-921. Without spongeforge it's working well any ideas?

```
[07:57:17] [Server thread/ERROR]: Encountered an unexpected exception
java.lang.NullPointerException
    at net.minecraft.world.storage.WorldInfo.doesKeepSpawnLoaded(WorldInfo.java:479) ~[bqo.class:?]
    at net.minecraft.world.World.onConstructed(World.java:334) ~[aqu.class:?]
    at net.minecraft.world.World.<init>(World.java:141) ~[aqu.class:?]
    at org.cyclops.evilcraft.core.world.FakeWorld.<init>(FakeWorld.java:32) ~[FakeWorld.class:?]
    at org.cyclops.evilcraft.core.world.FakeWorld.getInstance(FakeWorld.java:23) ~[FakeWorld.class:?]
    at org.cyclops.evilcraft.block.BoxOfEternalClosure.setVengeanceSwarmContent(BoxOfEternalClosure.java:161) ~[BoxOfEternalClosure.class:?]
    at org.cyclops.evilcraft.block.BoxOfEternalClosureConfig.onRegistered(BoxOfEternalClosureConfig.java:62) ~[BoxOfEternalClosureConfig.class:?]
    at org.cyclops.cyclopscore.config.ConfigHandler.loadConfig(ConfigHandler.java:89) ~[ConfigHandler.class:?]
    at org.cyclops.cyclopscore.config.ConfigHandler.handle(ConfigHandler.java:63) ~[ConfigHandler.class:?]
    at org.cyclops.cyclopscore.init.ModBase.preInit(ModBase.java:250) ~[ModBase.class:?]
    at org.cyclops.evilcraft.EvilCraft.preInit(EvilCraft.java:124) ~[EvilCraft.class:?]
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_65]
    at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_65]
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_65]
    at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_65]
    at net.minecraftforge.fml.common.FMLModContainer.handleModStateEvent(FMLModContainer.java:553) ~[forge-1.8-11.14.4.1563-universal.jar:?]
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_65]
    at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_65]
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_65]
    at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_65]
    at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74) ~[minecraft_server.1.8.jar:?]
    at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47) ~[minecraft_server.1.8.jar:?]
    at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322) ~[minecraft_server.1.8.jar:?]
    at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304) ~[minecraft_server.1.8.jar:?]
    at com.google.common.eventbus.EventBus.post(EventBus.java:275) ~[minecraft_server.1.8.jar:?]
    at net.minecraftforge.fml.common.LoadController.sendEventToModContainer(LoadController.java:212) ~[LoadController.class:?]
    at net.minecraftforge.fml.common.LoadController.propogateStateMessage(LoadController.java:190) ~[LoadController.class:?]
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_65]
    at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_65]
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_65]
    at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_65]
    at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74) ~[minecraft_server.1.8.jar:?]
    at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47) ~[minecraft_server.1.8.jar:?]
    at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322) ~[minecraft_server.1.8.jar:?]
    at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304) ~[minecraft_server.1.8.jar:?]
    at com.google.common.eventbus.EventBus.post(EventBus.java:275) ~[minecraft_server.1.8.jar:?]
    at net.minecraftforge.fml.common.LoadController.onPost(LoadController.java:49) ~[LoadController.class:?]
    at net.minecraftforge.fml.common.LoadController.distributeStateMessage(LoadController.java:119) ~[LoadController.class:?]
    at net.minecraftforge.fml.common.Loader.preinitializeMods(Loader.java:550) ~[Loader.class:?]
    at net.minecraftforge.fml.server.FMLServerHandler.beginServerLoading(FMLServerHandler.java:88) ~[FMLServerHandler.class:?]
    at net.minecraftforge.fml.common.FMLCommonHandler.onServerStart(FMLCommonHandler.java:355) ~[FMLCommonHandler.class:?]
    at net.minecraft.server.dedicated.DedicatedServer.func_71197_b(DedicatedServer.java:117) ~[po.class:?]
    at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:438) [MinecraftServer.class:?]
    at java.lang.Thread.run(Unknown Source) [?:1.8.0_65]
```

**Comment by rubensworks** [comment]
Probably because of some hooks that Sponge builds in, I'll see if I can do anything about it.

**Comment by Blue64** [comment]
@hxpx if you want to use the code tag:

```
Write
` ` `
it
Like
` ` `
This
(minus the spaces between the ` because if I remove the spaces here, it ends, then starts the [ code ] tag, and you won't be able to see the example)
It's the symbol next to the Tab Key & the 1 Key on a normal QWERTY Keyboard, Shift + ` = ~
Alternatively:
[ code ]
this
[ / code ]
minus spaces between the [ & ]
```

this allows for you to post the entire log in a neat & readable manner (complete with a sideways scroll feature instead of forcing it to wrap to a new line).
This also allows you to post the `` ` `<` ` `` & `` ` `>` ` `` symbols so that a specific section can be read "as is".

**Comment by hxpx** [comment]
@Blue64 thx i'm newbie on this website^^

**Comment by Blue64** [comment]
@hxpx `:)` I only wish someone had told me that info lol took me over a week to figure it out, other (encapsulating) formatting symbols include `*` or `_`
- x1 = _this_ 
- x2 = **this**
- x3 = **_this**_

to do the bullets, well:

``````
other (encapsulating) formatting symbols include ```*``` or ```_```
* x1 = *this* 
* x2 = **this**
* x3 = ***this***

to do the bullets, well:
``````

note the extra blank line, that breaks the bullet formatting and returns it to standard, otherwise it gets added to the end of the line above it (preview function in desktop view helps a lot too)

P. S. **_NEVER TYPE AN ENTIRE MESSAGE IN THIS FORMAT**_ lol it annoys people _way_ more than you'd believe `xD`

**Comment by ST-DDT** [comment]
@hxpx There is a nice link in the top right corner of the comment input field:
[Styling with Markdown is supported](https://guides.github.com/features/mastering-markdown/)

**Comment by Blue64** [comment]
@ST-DDT yes, but it's not there in Mobile View, and that also isn't a complete guide, it's more like a collection of references to more references. I didn't figure it out until I scoured the sub-pages of the Language's homepage, which I found by scouring GitHub's **entire** help section.

P. S. @hxpx avoid using `#` & `<>` & `[]` because those are all parsed out like the code tags.

```
# is a reference link
#316
This would link to issue # 316 in this GitHub Repository (which is this thread) if it wasn't within a Code Tag.
```

I have no idea how to remove a reference link once it's been posted lol

```
This is [a link to Google](http://www.Google.com).
```

This is [a link to Google](http://www.Google.com).

Idk what `<` & `>` are used for lol probably direct `HTML` code.

```
Oh yea, and for ~~Strike through~~ just surround the text with ~~
This is preferred to deleting text, because it may save confusion later.
```

~~Strike through~~

**Comment by rubensworks** [comment]
@hxpx Could you try the latest SpongeForge build? It might be fixed on their end (see SpongePowered/SpongeForge#454)
No EvilCraft update required.

**Comment by hxpx** [comment]
Trying it on lastest sponge version 1.8-1577-2.1-DEV-950
Here is **crashlog**

```
---- Minecraft Crash Report ----

WARNING: coremods are present:
  CCLCorePlugin (CodeChickenLib-1.8-1.1.2.115-universal.jar)
  FMLPlugin (InventoryTweaks-1.59-176.jar)
  CodeChickenCorePlugin (CodeChickenCore-1.8-1.0.5.34-universal.jar)
  SmartCorePlugin (SmartMoving-1.8-16.2.jar)
  NEICorePlugin (NotEnoughItems-1.8-1.0.5.82-universal.jar)
  LoadingPlugin (FastLeaveDecay-1.0.jar)
  SpongeCoremod (spongeforge-1.8-1577-2.1-DEV-950.jar)
  PlayerAPIPlugin (PlayerAPI-1.8-1.4.jar)
  CCCDeobfPlugin (unknown)
  TrashSlot (trashslot-mc1.8-2.0.12.jar)
  ThaumcraftLoader (Thaumcraft-1.8-5.0.3.jar)
  MalisisCorePlugin (malisiscore-1.8-1.2.0.jar)
  BOPLoadingPlugin (BiomesOPlenty-1.8-3.0.0.1505-universal.jar)
Contact their authors BEFORE contacting forge

// Why did you do that?

Time: 21.12.15 14:01
Description: Exception in server tick loop

java.lang.NullPointerException: Exception in server tick loop
    at net.minecraft.world.storage.WorldInfo.doesKeepSpawnLoaded(WorldInfo.java:477)
    at net.minecraft.world.World.onConstructed(World.java:334)
    at net.minecraft.world.World.<init>(World.java:141)
    at org.cyclops.evilcraft.core.world.FakeWorld.<init>(FakeWorld.java:32)
    at org.cyclops.evilcraft.core.world.FakeWorld.getInstance(FakeWorld.java:23)
    at org.cyclops.evilcraft.block.BoxOfEternalClosure.setVengeanceSwarmContent(BoxOfEternalClosure.java:161)
    at org.cyclops.evilcraft.block.BoxOfEternalClosureConfig.onRegistered(BoxOfEternalClosureConfig.java:62)
    at org.cyclops.cyclopscore.config.ConfigHandler.loadConfig(ConfigHandler.java:89)
    at org.cyclops.cyclopscore.config.ConfigHandler.handle(ConfigHandler.java:63)
    at org.cyclops.cyclopscore.init.ModBase.preInit(ModBase.java:250)
    at org.cyclops.evilcraft.EvilCraft.preInit(EvilCraft.java:124)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
    at java.lang.reflect.Method.invoke(Unknown Source)
    at net.minecraftforge.fml.common.FMLModContainer.handleModStateEvent(FMLModContainer.java:553)
    at sun.reflect.GeneratedMethodAccessor3.invoke(Unknown Source)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
    at java.lang.reflect.Method.invoke(Unknown Source)
    at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
    at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
    at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
    at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
    at com.google.common.eventbus.EventBus.post(EventBus.java:275)
    at net.minecraftforge.fml.common.LoadController.sendEventToModContainer(LoadController.java:212)
    at net.minecraftforge.fml.common.LoadController.propogateStateMessage(LoadController.java:190)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
    at java.lang.reflect.Method.invoke(Unknown Source)
    at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
    at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
    at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
    at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
    at com.google.common.eventbus.EventBus.post(EventBus.java:275)
    at net.minecraftforge.fml.common.LoadController.onPost(LoadController.java:54)
    at net.minecraftforge.fml.common.LoadController.distributeStateMessage(LoadController.java:119)
    at net.minecraftforge.fml.common.Loader.preinitializeMods(Loader.java:550)
    at net.minecraftforge.fml.server.FMLServerHandler.beginServerLoading(FMLServerHandler.java:88)
    at net.minecraftforge.fml.common.FMLCommonHandler.onServerStart(FMLCommonHandler.java:355)
    at net.minecraft.server.dedicated.DedicatedServer.func_71197_b(DedicatedServer.java:117)
    at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:438)
    at java.lang.Thread.run(Unknown Source)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- System Details --
Details:
    Minecraft Version: 1.8
    Operating System: Windows 7 (amd64) version 6.1
    Java Version: 1.8.0_65, Oracle Corporation
    Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
    Memory: 93924400 bytes (89 MB) / 341786624 bytes (325 MB) up to 6407585792 bytes (6110 MB)
    JVM Flags: 2 total; -Xincgc -Xmx6G
    IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
    FML: MCP v9.10 FML v8.0.99.99 Minecraft Forge 11.14.4.1563 29 mods loaded, 29 mods active
    States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored
    UCH mcp{9.05} [Minecraft Coder Pack] (minecraft.jar) 
    UCH FML{8.0.99.99} [Forge Mod Loader] (forge-1.8-11.14.4.1563-universal.jar) 
    UCH Forge{11.14.4.1563} [Minecraft Forge] (forge-1.8-11.14.4.1563-universal.jar) 
    UCH CodeChickenCore{1.0.5.34} [CodeChicken Core] (minecraft.jar) 
    UCH NotEnoughItems{1.0.5.82} [Not Enough Items] (NotEnoughItems-1.8-1.0.5.82-universal.jar) 
    UCH PlayerAPI{1.4} [Player API] (minecraft.jar) 
    UCH SmartCore{1.2.1} [Smart Core] (minecraft.jar) 
    UCH sponge{DEV} [Sponge] (spongeforge-1.8-1577-2.1-DEV-950.jar) 
    UCH Baubles{1.1.1.0} [Baubles] (Baubles-1.8-1.1.1.0.jar) 
    UCH Thaumcraft{5.0.3} [Thaumcraft] (Thaumcraft-1.8-5.0.3.jar) 
    UCH thaumicpalmistry{0.7} [Thaumic Palmistry] ([1.8] Thaumic Palmistry v0.7.jar) 
    UCH AfraidOfTheDark{1.8-1.1} [Afraid of the Dark] (AfraidOfTheDark-1.8-1.1.jar) 
    UCH anssrpg{v1.0.bd97738} [A Not So Simple RPG] (ANSSRPG-1.8-v1.0.bd97738.jar) 
    UCH BiomesOPlenty{3.0.0} [Biomes O' Plenty] (BiomesOPlenty-1.8-3.0.0.1505-universal.jar) 
    UCH CustomSpawner{3.4.0} [DrZhark's CustomSpawner] (CustomMobSpawner 3.4.0.jar) 
    UCH cyclopscore{0.3.0} [Cyclops Core] (CyclopsCore-1.8-0.3.0.jar) 
    UCH MoCreatures{8.0.1} [DrZhark's Mo'Creatures Mod] (DrZhark's Mo'Creatures Mod-8.0.1.jar) 
    UCH DungeonPack{1.8} [DungeonPack] (dungeonpack-1.8.jar) 
    UCE evilcraft{0.9.1} [EvilCraft] (EvilCraft-1.8-0.9.1.jar) 
    UCH FastLeaveDecay{1.0} [Fast Leave Decay] (FastLeaveDecay-1.0.jar) 
    UCH inventorytweaks{1.59-176-9318a76-dirty} [Inventory Tweaks] (InventoryTweaks-1.59-176.jar) 
    UCH IronChest{6.0.109.756} [Iron Chest] (ironchest-1.8-6.0.109.756-universal.jar) 
    UCH journeymap{5.1.3} [JourneyMap] (journeymap-1.8-5.1.3-unlimited.jar) 
    UCH malisiscore{1.8-1.2.0} [MalisisCore] (malisiscore-1.8-1.2.0.jar) 
    UCH malisisdoors{1.8-2.2.0} [Malisis' Doors] (malisisdoors-1.8-2.2.0.jar) 
    UCH recipehandler{1.8} [NoMoreRecipeConflict] (NoMoreRecipeConflict-0.3(1.8).jar) 
    UCH SmartMoving{16.2} [Smart Moving] (SmartMoving-1.8-16.2.jar) 
    UCH thaumictinkerer{${version}} [Thaumic Tinkerer] (ThaumicTinkerer-3.0-1.8-168 (1).jar) 
    UCH trashslot{2.0.12} [TrashSlot] (trashslot-mc1.8-2.0.12.jar) 
    Loaded coremods (and transformers): 
CCLCorePlugin (CodeChickenLib-1.8-1.1.2.115-universal.jar)
  codechicken.lib.asm.ClassHeirachyManager
  codechicken.lib.asm.RenderHookTransformer
FMLPlugin (InventoryTweaks-1.59-176.jar)
  invtweaks.forge.asm.ContainerTransformer
CodeChickenCorePlugin (CodeChickenCore-1.8-1.0.5.34-universal.jar)
  codechicken.core.asm.InterfaceDependancyTransformer
  codechicken.core.asm.TweakTransformer
  codechicken.core.asm.DelegatedTransformer
  codechicken.core.asm.DefaultImplementationTransformer
SmartCorePlugin (SmartMoving-1.8-16.2.jar)
  net.smart.core.SmartCoreTransformer
NEICorePlugin (NotEnoughItems-1.8-1.0.5.82-universal.jar)
  codechicken.nei.asm.NEITransformer
LoadingPlugin (FastLeaveDecay-1.0.jar)
  lumien.fastleavedecay.asm.ClassTransformer
SpongeCoremod (spongeforge-1.8-1577-2.1-DEV-950.jar)
  org.spongepowered.mod.asm.transformer.WorldGeneratorTransformer
  org.spongepowered.common.launch.transformer.SpongeSuperclassTransformer
PlayerAPIPlugin (PlayerAPI-1.8-1.4.jar)
  api.player.forge.PlayerAPITransformer
CCCDeobfPlugin (unknown)

TrashSlot (trashslot-mc1.8-2.0.12.jar)
  net.blay09.mods.trashslot.coremod.GuiContainerClassTransformer
  net.blay09.mods.trashslot.coremod.MinecraftClassTransformer
  net.blay09.mods.trashslot.coremod.InventoryTweaksClassTransformer
ThaumcraftLoader (Thaumcraft-1.8-5.0.3.jar)

MalisisCorePlugin (malisiscore-1.8-1.2.0.jar)
  net.malisis.core.util.chunkcollision.ChunkCollisionTransformer
  net.malisis.core.util.chunkblock.ChunkBlockTransformer
  net.malisis.core.renderer.transformer.MalisisRendererTransformer
BOPLoadingPlugin (BiomesOPlenty-1.8-3.0.0.1505-universal.jar)

    Profiler Position: N/A (disabled)
    Is Modded: Definitely; Server brand changed to 'fml,forge,SpongeForge'
    Type: Dedicated Server (map_server.txt)
```

**Comment by hxpx** [comment]
Here is **lastest.log**

``````
[14:00:51] [Server thread/INFO]: Starting minecraft server version 1.8
[14:01:03] [Server thread/ERROR]: Encountered an unexpected exception
java.lang.NullPointerException
    at net.minecraft.world.storage.WorldInfo.doesKeepSpawnLoaded(WorldInfo.java:477) ~[bqo.class:?]
    at net.minecraft.world.World.onConstructed(World.java:334) ~[aqu.class:?]
    at net.minecraft.world.World.<init>(World.java:141) ~[aqu.class:?]
    at org.cyclops.evilcraft.core.world.FakeWorld.<init>(FakeWorld.java:32) ~[FakeWorld.class:?]
    at org.cyclops.evilcraft.core.world.FakeWorld.getInstance(FakeWorld.java:23) ~[FakeWorld.class:?]
    at org.cyclops.evilcraft.block.BoxOfEternalClosure.setVengeanceSwarmContent(BoxOfEternalClosure.java:161) ~[BoxOfEternalClosure.class:?]
    at org.cyclops.evilcraft.block.BoxOfEternalClosureConfig.onRegistered(BoxOfEternalClosureConfig.java:62) ~[BoxOfEternalClosureConfig.class:?]
    at org.cyclops.cyclopscore.config.ConfigHandler.loadConfig(ConfigHandler.java:89) ~[ConfigHandler.class:?]
    at org.cyclops.cyclopscore.config.ConfigHandler.handle(ConfigHandler.java:63) ~[ConfigHandler.class:?]
    at org.cyclops.cyclopscore.init.ModBase.preInit(ModBase.java:250) ~[ModBase.class:?]
    at org.cyclops.evilcraft.EvilCraft.preInit(EvilCraft.java:124) ~[EvilCraft.class:?]
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_65]
    at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_65]
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_65]
    at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_65]
    at net.minecraftforge.fml.common.FMLModContainer.handleModStateEvent(FMLModContainer.java:553) ~[forge-1.8-11.14.4.1563-universal.jar:?]
    at sun.reflect.GeneratedMethodAccessor3.invoke(Unknown Source) ~[?:?]
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_65]
    at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_65]
    at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74) ~[minecraft_server.1.8.jar:?]
    at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47) ~[minecraft_server.1.8.jar:?]
    at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322) ~[minecraft_server.1.8.jar:?]
    at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304) ~[minecraft_server.1.8.jar:?]
    at com.google.common.eventbus.EventBus.post(EventBus.java:275) ~[minecraft_server.1.8.jar:?]
    at net.minecraftforge.fml.common.LoadController.sendEventToModContainer(LoadController.java:212) ~[LoadController.class:?]
    at net.minecraftforge.fml.common.LoadController.propogateStateMessage(LoadController.java:190) ~[LoadController.class:?]
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_65]
    at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_65]
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_65]
    at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_65]
    at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74) ~[minecraft_server.1.8.jar:?]
    at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47) ~[minecraft_server.1.8.jar:?]
    at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322) ~[minecraft_server.1.8.jar:?]
    at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304) ~[minecraft_server.1.8.jar:?]
    at com.google.common.eventbus.EventBus.post(EventBus.java:275) ~[minecraft_server.1.8.jar:?]
    at net.minecraftforge.fml.common.LoadController.onPost(LoadController.java:54) ~[LoadController.class:?]
    at net.minecraftforge.fml.common.LoadController.distributeStateMessage(LoadController.java:119) ~[LoadController.class:?]
    at net.minecraftforge.fml.common.Loader.preinitializeMods(Loader.java:550) ~[Loader.class:?]
    at net.minecraftforge.fml.server.FMLServerHandler.beginServerLoading(FMLServerHandler.java:88) ~[FMLServerHandler.class:?]
    at net.minecraftforge.fml.common.FMLCommonHandler.onServerStart(FMLCommonHandler.java:355) ~[FMLCommonHandler.class:?]
    at net.minecraft.server.dedicated.DedicatedServer.func_71197_b(DedicatedServer.java:117) ~[po.class:?]
    at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:438) [MinecraftServer.class:?]
    at java.lang.Thread.run(Unknown Source) [?:1.8.0_65]
[14:01:03] [Server thread/ERROR]: This crash report has been saved to ```
``````

**Comment by rubensworks** [comment]
@hxpx Could you subscribe to SpongePowered/SpongeForge#454 and test their new builds if they ask so?
I think those guys might be able to resolve the issue easier on their end.
I'm following the issue as well in case I should make any changes to EvilCraft.

**Comment by hxpx** [comment]
@rubensworks Ok thanks will watching this thread

**Comment by rubensworks** [comment]
Fixed at SpongePowered/SpongeForge#454
